configure.ac

```diff
@@ -548,12 +546,41 @@ if test -z "$ENABLE_SOS_TRUE"; then
 	fi
 fi
 
-dnl ldms-python API's
+dnl ldms-python API's and programs
+AX_WITH_PROG([CYTHON], [cython], [no])
+CYTHON_REQ_VER="0.28.5"
```

Comment:
How about "CYTHON_MIN_VER"? Makes it clearer that it is a minimum rather than a single required version.

Reply:
Sounds good. Somehow I couldn't think of "MIN" last night :laughing: